pkg/crc/validation/validation.go

```diff
@@ -98,6 +101,14 @@ func ValidateIPAddress(ipAddress string) error {
 	return nil
 }
 
+func ValidateURL(inputString string) bool {
+	u, err := url.ParseRequestURI(inputString)
+	if u.Scheme != "http" && u.Scheme != "https" && u.Scheme != "docker" {
+		return false
+	}
+	return err == nil
```

Comment:
If `err` is non-`nil`, `u` is invalid, so you can't check `u.Scheme` before checking `err` value